.github/workflows/releaseFromRun.yml

```diff
@@ -0,0 +1,54 @@
+name: Create release from Actions run
+on:
+  workflow_dispatch:
+    inputs:
+      run_id:
+        description: 'Actions run ID which to get artifacts from'
+        required: true
+        type: number
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      LISTS_DIR: lists
+      DEPENDENCIES_DIR: deps
+      GH_TOKEN: ${{ github.token }}
+    steps:
+      - name: Download artifacts
+        run: |
+          gh run download ${{ github.event.inputs.run_id }} \
+            --repo ${{ github.repository }} \
+            --dir "$LISTS_DIR" \
+            --pattern "list of*"
+          gh run download ${{ github.event.inputs.run_id }} \
+            --repo ${{ github.repository }} \
+            --dir "$DEPENDENCIES_DIR" \
+            --pattern "dependencies-*"
```

Comment:
I think we can also use `download-artifact` action here?

Reply:
true, but with CLI tool it's shorter